### DISCUSSION TITLE
Group Changes by subdirectory

### DISCUSSION THREAD

**Discussion by sinham-kkr** [renovatebot/renovate#32853] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

We are using renovate to automate updates to terraform projects.  A single repo may have many projects nested within sub folders. 

I would like to configure renovate to groups dependency updates within a project ( a single folder) 

how can i accomplish this?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Search for last questions with "additionalBranchPrefix" and "parentDir"
